packages/grid-pro/src/vaadin-grid-pro-inline-editing-mixin.js

```diff
@@ -120,15 +120,6 @@ export const InlineEditingMixin = (superClass) =>
       this.$.table.addEventListener('click', (e) => {
         const column = this.getEventContext(e).column;
         if (column && this._isEditColumn(column)) {
-          if (e.target.matches(':not([type=checkbox])')) {
```

Comment:
Why is this not needed anymore?

Reply:
You added the check originally [here](https://github.com/vaadin/web-components/commit/8d2118741ef05299da9ddab6e4f3fbfb13357b9c#diff-06adfcb7ef41b851ca16ad67fc0850660dd4333e07ae33e5a7baf684b2d72978R105-R112) to not prevent a user from interacting with checkboxes on click:

https://github.com/vaadin/web-components/assets/1222264/ad12d83d-f959-4de3-8f4e-7881684c0b26

Since this PR changes the implementation to no longer cancel the click event, checkboxes should work fine without the check. There's also a related [test case](https://github.com/vaadin/web-components/commit/8d2118741ef05299da9ddab6e4f3fbfb13357b9c#diff-b807015f53dd2ddece3613b9ff6fb3b4b0f6a03fd63f6c2236c883f15878aab7R122-R129) attached to the original PR.

Reply:
Looks like the preventDefault was used to prevent row selection when clicking on an editable column. However, I can see that such selection no longer occurs even without it, so something must have changed since then, which makes it safe to remove it.

Reply:
The row selection in Flow Grid is based on [active item changes](https://github.com/vaadin/flow-components/blob/244690872e4ea197ec9c4820f466194d175b8746/vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.ts#L182). That's why `preventDefault` also prevents it. This PR still prevents the active item change but without using `preventDefault`.

Reply:
I was testing selection with the web component while I should have with the Flow component. I see now. Thanks.